api/src/config.ts

```diff
@@ -0,0 +1,16 @@
+const getEnvString = (name: string): string => process.env[name] || '';
```

Comment:
this is tolerant of missing env var but `getEnvNumber` is not - it'll throw on the empty string I think. I'd vote to *not* tolerate missing values as that can just lead to hard to debug problems. Unless there's a good reason to have defaults in the code you don't want in the env but I don't see that below except for the server port.

One option is `getEnvString = (name: string, default: string) => ...` so the default is explicit and you can then throw if a default isn't provided and the string is missing or something.